test.js

```diff
@@ -358,7 +380,7 @@ test('err.code is 3', code, 3);
 test('err.code is 4', code, 4);
 
 test('timeout will kill the process early', async t => {
-	const err = await t.throws(m('delay', ['3000', '0'], {timeout: 1500}));
+	const err = await t.throws(m('delay', ['5000', '0'], {timeout: 1500}));
```

Comment:
Did this in an effort to fix the AppVeyor failure here: https://ci.appveyor.com/project/sindresorhus/execa/build/1.0.337

Not sure if this is a failure others see this type of failure regularly, but it passes locally and shouldn't be effected by my changes, so I'm not sure what else I should do about it.